app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageFragment.kt

```diff
@@ -82,7 +81,7 @@ import javax.inject.Inject
 class NewMessageFragment : Fragment() {
 
     private var _binding: FragmentNewMessageBinding? = null
-    private val binding get() = _binding!! // This property is only valid between onCreateView and onDestroyView
+    val binding get() = _binding!! // This property is only valid between onCreateView and onDestroyView
```

Comment:
I think we should avoid exposing the binding. Instead we can provide a method that returns the final data that we need to expose